libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -82,14 +83,17 @@ export const DealTicket = ({
 
   const order = watch();
 
-  watch((orderData) => {
-    const persistable = !(
-      orderData.type === OrderType.TYPE_LIMIT && orderData.price === ''
-    );
-    if (persistable) {
-      setPersistedOrder(orderData as DealTicketFormFields);
-    }
-  });
+  useEffect(() => {
+    const { unsubscribe } = watch((orderData) => {
+      const persistable = !(
+        orderData.type === OrderType.TYPE_LIMIT && orderData.price === ''
+      );
+      if (persistable) {
+        setPersistedOrder(orderData as DealTicketFormFields);
```

Comment:
on firefox the cursor does not move, but on chromium after first decimal decimal point is erased and cursor is moved at the start of the input. 

not sure where else to persist the order

Reply:
@MadalinaRaicu I was tinkering with this today and have a nice solution, lets chat tomorrow